staging/src/k8s.io/kubectl/pkg/cmd/create/create_service.go

```diff
@@ -388,22 +388,22 @@ func parsePorts(portString string) (int32, intstr.IntOrString, error) {
 
 	port, err := utilsnet.ParsePort(portStringSlice[0], true)
 	if err != nil {
-		return 0, intstr.FromInt(0), err
+		return 0, intstr.FromInt32(0), err
 	}
 
 	if len(portStringSlice) == 1 {
-		return int32(port), intstr.FromInt(int(port)), nil
+		return int32(port), intstr.FromInt(port), nil
```

Comment:
Is there are reason this is using the deprecated `intstr.FromInt`?

Perhaps something like this would work instead?  (or maybe there is a better way to get an int to an `IntOrString`)
```
return int32(port), intstr.FromInt32(int32(port)), nil
```

Reply:
I left it as a reminder (to myself) that it would be nice to decide on a type for ports across the code-base, _i.e._ in this case change `ParsePort` to return `int32` directly — but that does seem rather involved since it’s in k8s/utils.